packages/components/datepicker/src/components/osds-datepicker/osds-datepicker.tsx

```diff
@@ -147,6 +147,7 @@ export class OsdsDatepicker implements OdsDatepickerAttribute, OdsDatepickerEven
         ? this.minDate
         : undefined,
     });
+    this.hideSiblingsMonthDays();
```

Comment:
maybe this should be renamed as it does not necessarily hide things, something like `updateSiblingsMonthDaysVisibility`?